hx_lti_initializer/views.py

```diff
@@ -76,7 +77,13 @@ def initialize_lti_tool_provider(req):
 
 def create_new_user(username, user_id, roles):
     # now create the user and LTIProfile with the above information
-    user = User.objects.create_user(username, user_id)
+    # Max 30 length for person's name, do we want to change this? It's valid for HX but not ATG/FAS
```

Comment:
Are we using ```lis_person_sourcedid``` from the LTI launch for the username (i.e. the HUID)? HarvardX is using the ```lis_person_name_full```, which obviously won't work for us, since we *will* run into problems using that.